native-wheels-frontend/app/(tabs)/_layout.tsx

```diff
@@ -48,6 +38,25 @@ export default function TabLayout() {
           ),
         }}
       />
+        <Tabs.Screen
+            name="profile"
+            options={{
+                title: "Profile",
+                tabBarIcon: ({ color, focused }) => (
+                    <Ionicons name="person-circle-outline" size={24} color="black"
+                    />
```

Comment:
I would suggest to use the `<TabBarIcon>` component instead provided by the template and pass the name as a prop like the others, this will make sure it has the same style as the existing ones with same nice focused effect. 

For example it would be something like:
```typescript
<TabBarIcon
    name={focused ? "people-circle" : "people-circle-outline"}
    color={color}
/>
```

The same can be done for the settings icon tab below, as long as you use a settings icon from the same Ionicons pack which you can browse here: https://icons.expo.fyi/Index, for example it could be [this one](https://icons.expo.fyi/Index/Ionicons/settings) which also has a outline verrsion ensuring consistent styling between the icons.

Reply:
If you are really sweet you also change the car icon for Asbjørn to this:

```typescript
<TabBarIcon name={focused ? "car" : "car-outline"} color={color} />
```

So it also has a outline variant, making it all consistent and nice looking 😄